app/services/container_project_topology_service.rb

```diff
@@ -0,0 +1,36 @@
+class ContainerProjectTopologyService < TopologyService
+  include UiServiceMixin
+  include ContainerTopologyServiceMixin
+
+  @provider_class = ContainerProject
+
+  def build_topology
+    topo_items = {}
+    links = []
+
+    entity_relationships = { :ContainerProject => { :ContainerGroups => {
+                                                      :Containers          => nil,
+                                                      :ContainerReplicator => nil,
+                                                      :ContainerServices   => { :ContainerRoutes => nil }
+                                                    }
+                                                  }
+                           }
```

Comment:
@zeari are we taking advantage of the same logic of auto-disabling entities when we reach a limit? (PR https://github.com/ManageIQ/manageiq-ui-classic/pull/95) I think we should, because indeed we would like to show nodes and VMs here.